Robus/src/reception.c

```diff
@@ -168,16 +178,38 @@ void Recep_GetData(volatile uint8_t *data)
             {
                 MsgAlloc_Reset();
                 ctx.tx.status = TX_DISABLE;
+                Robus_SetNodeDetected(NETWORK_LINK_CONNECT);
+                Robus_SetVerboseMode(false);
             }
             else
             {
+                // check if we have already received the same message
+                if ((crc == last_crc) && (large_data_num > 0))
```

Comment:
Yes but it will take much more time to be checked as we dont have the crc in Luos big data reception, there we need to  compare the header + data of 2 messages

Reply:
and also to store always the whole previous message